task1.py

```diff
@@ -0,0 +1,17 @@
+# Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей. Дано a, b, c —
+# стороны предполагаемого треугольника. Требуется сравнить длину каждого отрезка-стороны с суммой
+# двух других. Если хотя бы в одном случае отрезок окажется больше суммы двух других, то треугольника
+# с такими сторонами не существует. Отдельно сообщить является ли треугольник разносторонним,
+# равнобедренным или равносторонним.
+
+
+side_a, side_b, side_c = map(int, input('Введите значение сторон треугольника через пробел ').split())
+
+if side_a < side_b + side_c and side_b < side_a + side_c and side_c < side_a + side_b:
+    if side_a == side_b == side_c:
+        print('Треугольник со сторонами', side_a, ',', side_b, 'и', side_c, '- равносторонний')
+    elif side_a == side_b or side_a == side_c or side_b == side_c:
+        print('Треугольник со сторонами', side_a, ',', side_b, 'и', side_c, '- равнобедренный')
+    else: print('Треугольник со сторонами', side_a, ',', side_b, 'и', side_c, '- разносторонний')
```

Comment:
Такая запись, хоть и допустима, но не принята
```suggestion
    else:
        print('Треугольник со сторонами', side_a, ',', side_b, 'и', side_c, '- разносторонний')
```